### PR TITLE
Fix NPE in ContentController#goBack()

### PR DESCRIPTION
**Pull request by mueller-ma**
Signed-off-by: mueller-ma <mueller-ma@users.noreply.github.com>

### CODE REVIEW

File: mobile/src/main/java/org/openhab/habdroid/ui/activity/ContentController.java [hunk 1]
```diff
@@ -398,11 +398,14 @@ public boolean canGoBack() {
...
-        if (mTemporaryPage instanceof FullScreenWebviewFragment) {
-            WebView webView = ((FullScreenWebviewFragment) mTemporaryPage).mWebview;
+        WebView webView = mTemporaryPage instanceof FullScreenWebviewFragment
+                ? ((FullScreenWebviewFragment) mTemporaryPage).mWebview : null;
+        if (webView != null) {
             if (webView.canGoBack()) {
                 webView.goBack();
                 return true;
+            } else {
+                webView.loadUrl("about:blank");
```
Comment (maniac103) [inline mobile/src/main/java/org/openhab/habdroid/ui/activity/ContentController.java]: What's the point of this? The WebView is going to be killed off when the fragment is removed anyway.

**Comment by maniac103** [comment]
Why not just add the null check to the existing `if ()` line?

File: mobile/src/main/java/org/openhab/habdroid/ui/activity/ContentController.java (hunk 1 above)
Comment (mueller-ma) [reply mobile/src/main/java/org/openhab/habdroid/ui/activity/ContentController.java]: If I go back only once I see the sitemap, but the fragment isn't killed. There are still log messages from HABPanel.

**Comment by mueller-ma** [comment]
Just personal prefference: I didn't want to call `((FullScreenWebviewFragment) mTemporaryPage).mWebview` twice.

**Pull request closed without merging**